src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -19,23 +23,63 @@ public CountryManager() {
         countryCodeMap = initCountryCodeMap();
     }
 
+    /**
+     * Checks if countryCode is a valid ISO3166 code.
+     *
+     * @param countryCode The country code.
+     * @return Whether countryCode is a valid ISO3166 code.
+     */
+    public static boolean isValidCode(String countryCode) {
```

Comment:
Would suggest changing this to isValidCountryCode, for clarity

Reply:
Changed.